client/src/components/feedbacks/comment.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import '../../style/feedback-comment.css';
+import PropTypes from 'prop-types';
+
+class FeedbackComment extends Component {
+  render() {
+    const { item } = this.props;
+    return (
+      <Container className="feedbackContainer">
+        <img src="/img/defAvatar.png" width="50px" height="50px" alt="defAvava" />
+        <div className="cooment">
+          <strong className="name">{item.userName}</strong>
+          <small className="date">{item.date}</small>
+          <hr className="hrStyle" />
+          <div>{item.description}</div>
+          <hr className="hrStyle" />
+          <div>
+
+            <span className="answer" title="answer to feedback">answer to</span>
+            <span className="mr-3 likeDislike">
+              <img src="/img/like.png" width="25px" height="25px" alt="like" title="like feedback" />
+              <small>{item.counterLike}</small>
+            </span>
+            <span className="mr-3 likeDislike">
+              <img src="/img/dislike.png" width="25px" height="25px" alt="dislike" title="dislike feedback" />
+              <small>{item.counterDislike}</small>
+            </span>
+
+            <span className="complaint">
+              <img src="/img/complaint.png" width="25px" height="25px" alt="complaint" title="complaint feedback" />
+            </span>
+
+          </div>
+        </div>
+      </Container>
+
+
+    );
+  }
+}
+
+FeedbackComment.propTypes = {
+  item: PropTypes.any.isRequired,
```

Comment:
I'm not sure if 'any' type is best one here. It can be passed non-suitable value mistakenly.

Fore sure, it can be 'object' type. And we still won't have any errors.

Reply:
This component is more powerful. We can pass an array or object into it.

Reply:
Yes, but we have no such requirement yet. 

Anyway, I agree with you. It is more flexible. At that rate we can use this:


```
FeedbackComment.propTypes = {
  item: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.array,
  ]).isRequired,
};
```

It will still work fine without any error messages.

Reply:
@VanderbiGit you can't pass array in this `item`. In this case there will be errors  `property userName doesn't exist in Array`
```
<strong className="name">{item.userName}</strong>
          <small className="date">{item.date}</small>
```
If you want to add possibility to pass array you need to add condition like
```javascript
if (Array.isArray(item)) {
  // your return code with for 
} else {
  // your current return code
}
```
But I agree with you that `any` type is well enough.